### PR TITLE
Fix active param count

### PR DESCRIPTION
**Pull request by j321m**
# Description
This pr fixes counting model parameters. Instead of getting count from torch model, it calculates from args using equations (brrr math). There are model configurations that aren't handled, in such case 1 is returned as model param count. This is because we don't want to kill new modules and 1 schould be strange enough to be recognised as wrong. Calculating from torch is still there and printed as sanity check. Main feature is fixing token to param ratio, and token_per_active x-axis on loss charts in neptune.

What I'm not sure about are the names of the new args variables, args that are logged to neptune. I tried to be descriptive, but shout out if sth is confusing.

# Checklist
   - [x] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [x] Are all variables set with logical defaults that are backward compatible?
   - [x] [I attached link to neptune if it was necessary](https://app.neptune.ai/o/pmtest/org/llm-random/runs/details?viewId=standard-view&detailsTab=metadata&shortId=LLMRANDOM-19203&type=run&compare=IwTgTADAzEA&lbViewUnpacked=true&sortBy=%5B%22sys%2Fcreation_time%22%5D&sortFieldType=%5B%22datetime%22%5D&sortFieldAggregationMode=%5B%22auto%22%5D&sortDirection=%5B%22descending%22%5D&query=((%60sys%2Ftags%60%3AstringSet%20CONTAINS%20%22param_count_fix_test%22)))
   - [x] Have you successfully executed the linter and tests?
   - [x] Are all functions concise and don't require splitting?
   - [x] Is there documentation for arguments and complex logic?
   - [x] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [x] Are all functions placed in appropriate files?
   - [x] Is the PR name meaningful and descriptive?
   - [x] Is PR description provided, or unnecessary?
  
Other questions:
   - [ ] If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters
   - [x] If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)
   - [ ] If there is a need for a second reviewer I requested additional review 

# Reviewer's checklist:
   - [ ] Does the Neptune link work and does it actually compare the situation before and after the PR?
   - [ ] Do I understand the code?
   - [ ] Do I think another reviewer is not needed or I commented that this PR need another reviewer?

### CODE REVIEW

File: lizrd/support/misc.py [hunk 1]
```diff
@@ -17,26 +18,176 @@ def get_n_learnable_parameters(model):
...
+    if model_configuration is None:
+        return (
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+            1,
+        )  # something that won't kill experiments, but is obviously not true
```
Comment (madragonse) [inline lizrd/support/misc.py:130]: can you explain differently this result case?

**Review by madragonse — CHANGES_REQUESTED** [review]
Generally, looks good, altho haven't checked the math

File: lizrd/support/logging.py [hunk 2]
```diff
@@ -495,22 +498,79 @@ def report_plotly(
...
+    print(f"Model total parameters from equation:\t{args.model_n_params:_}")
+    print(f"Model nonembedding parameters:\t\t{args.model_n_nonembedding_params:_}")
+    # print(f"debug difference:\t{all_model_params_calculated_directly_from_torch - args.model_n_params}")
+    # print(f"embedding_params: {embedding_params}")
+    # print(f"all_layer_norm_params: {all_layer_norm_params}")
+    # print(f"all_attention_params: {all_attention_params}")
+    # print(f"all_ff_total_params: {all_ff_total_params}")
+    # print(f"all_ff_active_params: {all_ff_active_params}")
+    # print(f"all_router_params: {all_router_params}")
+    # print(f"head_params: {head_params}")
```
[the remark this answers is not in the document]
Comment (j321m) [reply lizrd/support/logging.py]: ok

**Comment by madragonse** [comment]
ty, approved

**Pull request merged**